packages/sdk/src/actions/acceptOffer.ts

```diff
@@ -8,7 +8,8 @@ import {
   refreshLiquidity,
 } from '../utils'
 import { WalletClient } from 'viem'
```

Comment:
Are these unused?

Reply:
<img width="619" alt="Screenshot 2024-01-16 at 9 10 21 AM" src="https://github.com/reser
<img width="458" alt="Screenshot 2024-01-16 at 9 10 38 AM" src="https://github.com/reservoirprotocol/reservoir-kit/assets/97770763/58ae6665-4fdf-4ab6-af9e-4f843c433e43">
voirprotocol/reservoir-kit/assets/97770763/d5ee912c-106b-458d-a35e-a0b30d465e9c">

Reply:
Seem to be used.